docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
We have versioning for this exact purpose.

Not really appart for keeping it simple.